app/controllers/sessions_controller.rb

```diff
@@ -32,4 +24,16 @@ def destroy
   def get_user
     @user = User.find_by email: params[:session][:email]
   end
+
+  def user_active
```

Comment:
- rename to `is_activated?`
- the method needs a parameter
- you can place this to `user.rb` model also